DBMS module/flow_sequence_diag.md

```diff
@@ -1,6 +1,6 @@
 # Data Flow Diagram
 
-<img width="700" height="535" src="/images/flow-sequensce.png"> 
+<img width="700" height="535" src="../images/flow-sequensce.png"> 
```

Comment:
@Antsthebul its working and the image is visible I think so need to change here

Reply:
@IIITM-Jay We should better deal this in another issue.